.github/workflows/pythonapp.yml

```diff
@@ -17,3 +17,7 @@ jobs:
       run: |
         python -m pip install --upgrade pip
         pip install -r requirements.txt
+    - name: Run checks
+      run: |
+        pip install mypy
+        python checks.py --since origin/master --typing
```

Comment:
This causes the CI to test the files modified in the current branch, we can update this with a commit hash or a tag instead so it will actually test something for the master branch.

Reply:
So for instance, if we add new code to an existent file, `mypy` will check the whole file or only the new code?

Reply:
`mypy` will check the whole file.

I would prefer running it against all the code, but for now this should do the trick.

Reply:
Ah great. Would be better to solve all the current code base `mypy` errors related with "... no type hints or library stubs" in this PR? Only to avoid these errors on the following PR's.

Reply:
We can fix them when we find them, no need to rush it here in this case I think

Reply:
You are right. Will be better do it as we move on :+1: